apis/swagger.yml

```diff
@@ -1964,6 +1964,10 @@ definitions:
       QuotaID:
         type: "string"
         description: "set disk quota by specified quota id, if id < 0, it means pouchd alloc a unique quota id"
+      NetPriority:
```

Comment:
NetPriority seem a property of network endpoint. A container may have many endpoints, we may apply different Priorities to different endpoints. Can we consider the situation?

Reply:
Since we not implement in sever, so we just add a flag called `NetPriority` to prepare